content/rest-api/authentication.md

```diff
@@ -0,0 +1,253 @@
+# Authentication
+
+For security reasons, to access the API, you will need to be authenticated.
+To authenticate your API calls, the PIM will provide you a pair of client ID and secret. [See here to discover how](#client-idsecret-generation).
+
+Client IDs allow two things:
+- to ask for a token for a specific API user,
+- to refresh an expired token.
+
+This means that you will also need to [create an API user](#api-user-creation) in order to get this token. 
+
+Once you get its username and password, together with the client ID and the secret, you will eventually be able to generate a token for the API. This is the _key_ to finally access the API. See [here for more details](#token-generation).
+
+## Client ID/secret generation
+
+Depending on your PIM version, there are two different ways to generate these client IDs/secrets:
+- Proud user of a v4 PIM, you can follow [these simple steps](#starting-from-the-v4-of-the-pim).
+- Using a PIM v2.x or v3.x? [Follow the instructions right here.](#with-a-v2x-and-3x-pim)
+- Still using the v1.7? [Here you go!](#with-a-17-pim)
+
+### Starting from the v4 of the PIM
+Starting from the v4 of the PIM, there is now one unique and streamlined way to generate your client ID and secret. See by yourself!
+
+1. Log into your favorite PIM.
+1. Navigate to the `System/Connections` menu.
+1. Click on the `Create` button.
+1. Input a label for your connection, `ERP` for example.
+1. Select a type flow. [_Don't know what it is? More info here._](https://help.akeneo.com/pim/articles/manage-your-connections.html#choose-your-flow-type)
+1. Click on the `Save` button.
+
+Congrats! You officially created your very first connection to the PIM. :tada:
+
+You can now find the client ID and secret in the `Credentials` section of the connection you've just created.
+
+[Not comfortable with the **Connection** notion? We explain everything right here!](https://help.akeneo.com/pim/articles/what-is-a-connection.html)
+
+::: info
+In case your secret leaked, you can easily revoke it and generate a new one for your connection by clicking on the dedicated icon, on the right side of the secret.
+:::
+
+### With a v2.x and 3.x PIM
+
+In 2.x or 3.X, you will need to create what we called an `API connection`. 
+
+1. Log into your favorite PIM.
+1. Navigate to the `System/API connections` menu. 
+1. Click on the `Create` button. 
+1. Input a label for your connection, `ERP connection` for example.
+1. Click on the `Save` button.
+
+The PIM automatically generates a client ID and secret, that you will find in the API connections grid.
+
+::: info
+In case your secret leaked, you can revoke an API connection by clicking on the `Revoke` button, in the `API connections` entry you will find in the `System` menu.  
```

Comment:
```suggestion
In case your secret leaked, you can revoke an API connection by clicking on the `Revoke` button. It is located in the `API connections` entry you will find under the `System` menu.  
```